### ISSUE TITLE
Multiple genre tags from local backend are concatenated into one string.

### ISSUE THREAD

**Issue by crocket** [mopidy/mopidy#1757]
I installed and ran mopidy-2.2.2 on gentoo linux.

On mpd, multiple genre tags are separated properly.
On mopidy, multiple genre tags are concatenated into one string with semicolons between each genre.

**Comment by kingosticks** [comment]
Our MPD frontend uses [concat_multi_values](https://github.com/mopidy/mopidy/blob/develop/mopidy/mpd/translator.py#L133) to produce a single line from our lists which I assume mimics mpd's behaviour (at some point in time but maybe no longer).  However, [genre is not a list](https://docs.mopidy.com/en/latest/api/models/?highlight=genre#mopidy.models.Track), it's a single string. And we convert it from a list to that delimited string [here](https://github.com/mopidy/mopidy/blob/develop/mopidy/audio/tags.py#L101). 

So fixing this will require a change to our `Track` model also.

**Comment by crocket** [comment]
It seems TAG_GENRE, TAG_TITLE, TAG_ORGANIZATION, comment, TAG_LOCATION, and TAG_COPYRIGHT are joined into a single string.

Would it not be better to convert them back to lists?

**Comment by kingosticks** [comment]
For genre, yes, it makes sense. For title we are probably OK as we are. Personally I don't think there's an obvious reason to support multiple separate titles so why introduce that confusion to every client? And the others are really just hacked into the comment because we have no motivation to expose them.

**Comment by crocket** [comment]
Yes, it's good to focus just on genre for now. When I want to see multiple titles, I will create a new issue.
